### DISCUSSION TITLE
Chart 鼠标移动时候自定义提示

### DISCUSSION THREAD

**Discussion by ArgoZhang** [dotnetcore/BootstrapBlazor#4087] [Q&A] [answered]
Chart 组件如何支持自定义回调方法

**Comment by AiYuZhen** [comment]
https://www.chartjs.org/docs/master/api/interfaces/Plugin.html
是不是可以考虑封一个专用于回调 cs 的插件，然后在各节点回调 cs
应该 Chart 所有涉及回调的大部分都是在 插件 部分吧
无法用纯 cs 完成的功能就在对应的回调内再调用 js 方法

**Reply by ArgoZhang** [reply]
link issue 4090
link pr 4091

**Comment by ArgoZhang** [comment, marked as answer]
增加 `BootstrapBlazor.Chart` 全局类，自定义脚本中可以通过 `getOptionsById` 获得当前 `Chart` 组件的配置，再次进行扩展，具体例子可以看 

https://www.blazor.zone/chart/line#CustomTooltip
